corehq/apps/hqadmin/dbaccessors.py

```diff
@@ -17,6 +17,15 @@ def get_all_forms_in_all_domains():
 
 
 def get_number_of_forms_in_all_domains():
+    """
+    Return number of non-error forms (but incl. logs) total across all domains
+    specifically as stored in couch.
+
+    (Can't rewrite to pull from ES or SQL; this function is used as a point
+    of comparison between row counts in other stores.)
+
+    """
+
     return XFormInstance.view('hqadmin/forms_over_time').one()['value']
```

Comment:
Do you mean we can kill this whole [db_comparisons](https://github.com/dimagi/commcare-hq/blob/01464841f387741fa8c6b6f58534087230618c90/corehq/apps/hqadmin/views.py#L354-L396) feature?